### ISSUE TITLE
No rsudp_settings.json file appears.

### ISSUE THREAD

**Issue by bschellhaas** [raspishake/rsudp#19]
I have unzipped the software, ran the .bat file, however, no files appear in the .config/rsudp. There didn't seem to be any errors in the install prompt, however, when I tried to run the program, this error occurred. I have also tried the advanced install, however, the same outcome happens. Any advice would be appreciated.
![rsudperror](https://user-images.githubusercontent.com/58160447/108721229-61ae1c80-74e7-11eb-815b-8c0e5f585368.png)

**Comment by bschellhaas** [comment]
I tried recreating the .json file found in the install tutorial, but the same FileNotFoundError happened. Still need help.

**Comment by iannesbitt** [comment]
Hi @bschellhaas:
What happens when you run the command `rs-client -i`?

**Comment by alihanks** [comment]
I'm having the same issue. When trying to perform a new installation of rsudp, it states that the rsudb_settings.json file is not found. I don't see an example .json file anywhere in the installation tutorial anymore, either.

**Comment by kylebrill** [comment]
I can confirm that when installing using directions in [2.2.2](https://raspishake.github.io/rsudp/installing.html#advanced-users) under the Windows install instructions, that navigating to my Miniconda environment directory: `%USERPROFILE%\Miniconda3\envs\rsudp\`, going into the `Scripts` directory, and running `rs-client.exe -i` did create the **rsubd_settings.json** file in my `%USERPROFILE%\.config\rsudp` directory.
Running `rs-settings.exe` in the same Miniconda environment directory opened an editor to work with the json file.
Might want to update the instructions now that the "Easy Way" doesn't seem to be working anymore?
[Download](https://github.com/raspishake/rsudp/releases/download/1.1.0/rsudp.zip) results in a `Not Found`.

**Comment by iannesbitt** [comment]
Hi @kylebrill, I've corrected the link endpoint for [Download](https://github.com/raspishake/rsudp/archive/refs/heads/master.zip). Sorry, this was broken by a github code change that I did not pick up on. I added manual instructions in case it breaks again.

I don't do the development for the Windows scripts, so I will have to look into why this is happening and let our Windows guy know. Thanks for testing, good to know that the `-i` flag still works at least.
